runtime/vm/vmbootlib.c

```diff
@@ -650,6 +650,9 @@ classLoaderRegisterLibrary(void *voidVMThread, J9ClassLoader *classLoader, const
 					if (NULL != libraryPtr) {
 						*libraryPtr = newNativeLibrary;
 					}
+#ifdef J9VM_OPT_JAVA_OFFLOAD_SUPPORT
+					newNativeLibrary->doSwitching = validateLibrary(javaVM, logicalName, newNativeLibrary->handle, JNI_TRUE);
+#endif /* J9VM_OPT_JAVA_OFFLOAD_SUPPORT */
```

Comment:
This should use `#if defined()`:
```c
#if defined(J9VM_OPT_JAVA_OFFLOAD_SUPPORT)
					newNativeLibrary->doSwitching = validateLibrary(javaVM, logicalName, newNativeLibrary->handle, JNI_TRUE);
#endif /* defined(J9VM_OPT_JAVA_OFFLOAD_SUPPORT) */
```
See also lines 717, 721.

Reply:
Updated.